### DISCUSSION TITLE
可以在此板块讨论任何相关话题

### DISCUSSION THREAD

**Discussion by liangjingkanji** [liangjingkanji/Net#34] [Announcements]
当然你也可以加群讨论

**Comment by nuclearenergyant** [comment]
在没有网络的情况下，同步请求页面例子会出现报错

**Reply by liangjingkanji** [reply]
同步请求崩溃正常, 除非你catch住. 不过例子应该catch一下

**Comment by JackiePenghe** [comment]
以后会不会加入自动重试相关的API，比如失败后自动重试，重试次数之类的API

**Reply by liangjingkanji** [reply]
自带RetryInterceptor支持这个功能, 但是我建议你别自动重试, 失败就是失败显示失败重试缺省页就好了, 否则你会导致用户卡顿非常久时间而且你还不能发现问题所在, 这是愚蠢的做法

**Comment by UiNiNex** [comment]
最近项目总是出现请求资源id错误，查看日志是不包含http或https的请求，目前没找到触发原因
